flag_impl.go

```diff
@@ -57,6 +81,8 @@ type FlagBase[T any, C any, VC ValueCreator[T, C]] struct {
 
 	Config C // Additional/Custom configuration associated with this flag type
 
+	NonDuplicated bool // whether this flag can be duplicated on the command line
```

Comment:
I feel weird about a bool type name that starts with a negation, but I understand we want the default behavior to be to allow multiple instances. What do you think about:

- `StrictOnce`
- `StrictlyOnce`
- `Once`
- `OnlyOnce`

?

Reply:
* `Repeatable`

?

Reply:
I think OnlyOnce is good since by default it would be false.